app/src/androidTest/java/org/simple/clinic/TestClinicApp.kt

```diff
@@ -3,6 +3,7 @@ package org.simple.clinic
 import android.app.Application
 import com.tspoon.traceur.Traceur
```

Comment:
Can `patient_id_card` icon not be a VectorDrawable?

Reply:
We tested it out first, and the vector asset had some issues being imported. I did inform Akshay, but I don't know if he managed to get time to correct it.

I'd like to leave it as static images for now, and get the vector in later.

Reply:
okay